Quick-Settings/app/src/main/res/values/strings.xml

```diff
@@ -18,10 +18,16 @@
 <resources>
     <string name="app_name">Quick Settings Sample</string>
     <string name="tile_label">Quick Settings Sample</string>
-    <string name="app_greeting">To open the notification shade, swipe down from the top of the phone.
-        Expand the widget on the right side and then click <b>Edit</b>.
+    <string name="app_greeting"><b>Manual</b>\nOpen the notification shade to
```

Comment:
Does this `<b>` work without `Html.fromHtml`?

Also, the content should be surrounded with `<![CDATA[` ... `]]>` to use `<` in XMLs.